src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -1,35 +1,63 @@
 .pf-c-notification-badge {
-  --pf-c-notification-badge--after--BorderColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-notification-badge--after--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-notification-badge--Color: var(--pf-global--ZIndex--sm);
+  --pf-c-notification-badge--child--ZIndex: var(--pf-global--ZIndex--sm);
+
+  // Notification badge
+  --pf-c-button--c-notification-badge--MarginRight: calc(var(--pf-global--spacer--xs) * -1);
+  --pf-c-button--c-notification-badge--MarginLeft: calc(var(--pf-global--spacer--xs) * -1);
+
+  // ::after border treatment
+  --pf-c-notification-badge--after--BorderColor: var(--pf-global--BackgroundColor--dark-200);
+  --pf-c-notification-badge--after--BorderRadius: var(--pf-global--BorderRadius--md);
   --pf-c-notification-badge--after--BorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-notification-badge--after--ZIndex: var(--pf-global--ZIndex--xs);
   --pf-c-notification-badge--after--Top: 0;
   --pf-c-notification-badge--after--Right: 0;
-  --pf-c-notification-badge--after--Width: calc(var(--pf-global--spacer--sm) + var(--pf-c-notification-badge--after--BorderWidth) + var(--pf-c-notification-badge--after--BorderWidth));
-  --pf-c-notification-badge--after--Height: calc(var(--pf-global--spacer--sm) + var(--pf-c-notification-badge--after--BorderWidth) + var(--pf-c-notification-badge--after--BorderWidth));
+  --pf-c-notification-badge--after--Bottom: 0;
+  --pf-c-notification-badge--after--Left: 0;
+  --pf-c-notification-badge--after--Width: auto;
```

Comment:
is there any point to this ? if you dont want to remove it because of breaking changes could you leave a comment next to it. same as below